bot/exts/evergreen/tic_tac_toe.py

```diff
@@ -276,6 +276,9 @@ async def tic_tac_toe(self, ctx: Context, opponent: t.Optional[discord.User]) ->
             )
         self.games.append(game)
         if opponent is not None:
+            if opponent.bot:  # check whether the opponent is a bot or not
+                return await ctx.send("You can't play Tic-Tac-Toe with bots!")
```

Comment:
```py
async def tic_tac_toe(self, ctx: Context, opponent: t.Optional[discord.User]) -> None:
```
The return type hint here is None, yet you are returning a Message object.
```suggestion
            if opponent.bot:  # check whether the opponent is a bot or not
                await ctx.send("You can't play Tic-Tac-Toe with bots!")
                return
```